src/main/scala/kamon/newrelic/LibraryVersion.scala

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package kamon.newrelic
+
+import java.util.Optional
+
+object LibraryVersion {
+  val version: String =
+    Optional.ofNullable(classOf[LibraryVersion].getPackage.getImplementationVersion)
+      .orElse("UnknownVersion");
+}
+
+private class LibraryVersion
```

Comment:
I couldn't figure out how to get the class of the object, so I added this little dingle

Reply:
Decent hack!